Updated brazilian portuguese translation

Felipefpl opened the pull request:
Included the latest changes.

Felipefpl committed:
Updated brazilian portuguese translation

Included the latest changes.

hrydgard commented on assets/lang/pt_BR.ini:
```diff
@@ -23,54 +23,41 @@
-%1: Attempt failed = %1: Attempt failed
-%1: Attempt started = %1: Attempt started
 %d achievements, %d points = %d conquistas, %d pontos
+%1: Leaderboard attempt started = %1: A tentativa da tabela de classificação começou
+%1: Leaderboard attempt failed = %1: A tentativa da tabela de classificação falhou
```
Hm, looks like you switched to the old keys here? I don't think you started from the latest?

hrydgard replied:
If you go look in the latest:

https://github.com/hrydgard/ppsspp/blob/master/assets/lang/en_US.ini#L27

You see that I've removed "Leaderboard" from these two lines for example, because it took up too much space. In your change though, Leaderboard is still there on the left side of the = sign.

Felipefpl replied:
Fixed. Thank you for showing my mistake.

hrydgard replied:
I think what happened is that your branch was from an old master in your repo.

Felipefpl replied:
nope. it's not this, i made the sync today.

Felipefpl committed:
Fixed previous mistakes

My previous submission was incorrect because several strings were missing although i'm sure i got the english file for comparison correctly. Weird...

Thanks @hrydgard for showing the problem to me.
